components/eam/bld/configure

```diff
@@ -1062,25 +1062,14 @@ if ($print>=2) { print "Using $pbl_pkg PBL scheme.$eol"; }
 
 # Set default
 my $rad_pkg = 'rrtmg';
-if ($phys_pkg =~ m/^ideal$|^adiabatic$|^cam[34]$/) {
-    $rad_pkg = 'camrt';
-}
 
 # Allow the user to override the default via the commandline.
 if (defined $opts{'rad'}) {
     $rad_pkg = lc($opts{'rad'});
 }
 
 # consistency checks...
-
-if ($rad_pkg eq 'camrt') {
-
-    # The camrt radiation doesn't work with the modal aerosols
-    if ($chem_pkg =~ /_mam/) {
-        die "configure ERROR: radiation package: $rad_pkg is not compatible\n".
-            "                 with aerosol package $chem_pkg\n";
-    }
-} elsif ($rad_pkg eq 'rrtmg' or $rad_pkg eq 'rrtmgp') {
+if ($rad_pkg eq 'rrtmg' or $rad_pkg eq 'rrtmgp') {
 
     # The rrtmg package doesn't work with the CAM3 prescribed aerosols
```

Comment:
You could probably delete this block if you want to up your "deleted lines" score

Reply:
I'm not sure whether or not to remove this here or in a separate PR that removes other mentions of cam3 and cam4 values for `-phys`. Looking a little deeper, the only values of `-phys` that are used by EAM compsets in `components/eam/cime_config/config_component` are `default`, `ideal`, and `adiabatic`. I.e., these appear to be the only ways that EAM is used. So it appears as though ALL of this logic surrounding cam3/cam4/cam5 can go away, and greatly simplify the configure script. Thoughts? Pinging @rljacob for opinions?

Reply:
You should ask @wlin7 who has been looking at simplifying the configure script.